src/components/submission-common.js

```diff
@@ -199,8 +199,8 @@ const parseMethodString = methodString => {
   const methodNameParts = methodString.split('.')
   return {
     workflow: methodNameParts[0],
-    variable: methodNameParts[methodNameParts.length - 1],
-    call: methodNameParts.length === 3 ? methodNameParts[1] : ''
+    task: methodNameParts.length === 3 ? methodNameParts[1] : '',
```

Comment:
So "call" is actually more accurate here... and would cover the concept of a workflow OR a task, so might even be better in the UI too

Reply:
ah, ok. I'll change that back.